src/components/ActionButton.native.tsx

```diff
@@ -1,14 +1,25 @@
 import React from 'react'
-import { ActivityIndicator, StyleProp, TouchableOpacity, ViewStyle, StyleSheet } from 'react-native'
+import { ActivityIndicator, StyleProp, ViewStyle } from 'react-native'
 import { Text } from './Text.native'
+import tw from 'tailwind-react-native-classnames'
+import styled from 'styled-components/native'
+import themeSpecific from '../styled-helpers/themeSpecific'
+
+const Button = styled.TouchableOpacity`
+  ${tw`p-2 my-4 items-center rounded-lg w-6/12` as Record<string, string>}
+  ${themeSpecific({
+    dark: tw`bg-white`,
+    light: tw`bg-black`,
+  })}
```

Comment:
This looks good. 

I'm still wrapping my head around how the theme gets passed to this. Where do the props to `themesSpecific` come from?

Reply:
Thank you!